lib/chore_runner/downloads.ex

```diff
@@ -0,0 +1,26 @@
+defmodule ChoreRunner.Downloads do
+  alias ChoreRunner.Downloads.StorageService
+
+  def create_download(filename, opts), do: StorageService.save_file(filename, opts)
+
+  def list_downloads do
+    with {:ok, local_downloads} <- StorageService.list_files(),
```

Comment:
If you want, I think you could simplify this a bit by using `:rpc.multicall([node() | Node.list()], StorageService, :list_files, [])` so you only have to worry about the result of one function and don't need to concat the local/remote files later

Reply:
I prefer not going through `:rpc` on the local node